packages/sdk-codegen/src/testUtils/testUtils.ts

```diff
@@ -59,17 +60,20 @@ export interface ITestConfig {
   testSection: IKeyAny
 }
 
+const homeToRoost = '../../../../'
+
+export const getRootPath = () => path.join(__dirname, homeToRoost)
```

Comment:
TIL `__{dir,file}name` - python version is `__file__` - I find the choice of leading and trailing double underscores between languages ... entertaining